pkg/engine/avsets.go

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+package engine
```

Comment:
slight preference for not abbreviating availability sets in the filename (`availabilitysets.go`)

Reply:
Agreed. It's a better name.